cmd/manager/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
```

Comment:
Rather than adding tests to main I think it would make sense to use the integration suite https://github.com/app-sre/deployment-validation-operator/tree/master/integration. This should reduce coupling between tests and implementation.